OpenRA.Mods.Common/Traits/Player/PlayerStatistics.cs

```diff
@@ -45,8 +37,10 @@ public int Experience
 			}
 		}
 
+		// Low resolution (every 30 seconds) record of earnings, covering the entire game
 		public List<int> EarnedSamples = new List<int>(100);
```

Comment:
Rename to IncomeSamples? Not sure if its worth to keep the naming consistent with the rest.

Reply:
This really is sampling the earned resources. Income is calculated from the difference of the first and last earning samples.

Reply:
Maybe I'm just confused but we are only adding `Income` to `EarnedSamples` and then its displayed in the `IncomeGraph`. The calculation of `Income` is not related to this property except its stored in this list.